### ISSUE TITLE
[ChinaAzure/Mooncake]Microsoft graph SDK for powershell cannot be used cause of required application cannot be found in AAD tenant.

### ISSUE THREAD

**Issue by Icybiubiubiu** [microsoftgraph/msgraph-sdk-powershell#1032]
Our environment is ChinaAzure/Mooncake and we follow this document to use Microsoft Graph ADK for powershell.
https://docs.microsoft.com/en-us/graph/powershell/get-started

when use this command to connect to Mooncake, we always this AADSTS700016 error while it works well in public Azure.
![image](https://user-images.githubusercontent.com/61815812/151115845-274dfe20-12ea-48e1-9bdc-8f58c452450c.png)

I have checked the user I used, it is Global Admin in AAD.
And when I want to install SP for this app, it always failed:
![image](https://user-images.githubusercontent.com/61815812/151116477-d74ed0e7-57fd-4a8c-9201-c1d5c48cec30.png)

Could you please configure this default/first party app for tenants in Mooncake?

**Comment by Icybiubiubiu** [comment]
@jasonjoh , Could you please help to give some comments about this issue?

**Comment by Icybiubiubiu** [comment]
I have resolved this by registered my own app...lalalalalala~
Will close this issue.

**Comment by maisarissi** [comment]
Hello @Icybiubiubiu .
We have added support for national clouds through this PR https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/425
Apps don't replicate to China cloud; we need to add a special appId registration for mooncake.

As a workaround, it's necessary to register your own app with a certificate in https://portal.azure.cn.
Once done, call Connect-MgGraph with -Environment set to China using the app registration you made. 
```powershell
Connect-MgGraph -Environment China -AppId $CN_App_Id -TenantId $CN_Tenant_Id -CertificateThumbprint $Cert_Thumbprint
```
Call any command with -Debug and inspect the request url before the response. It should be "https://microsoftgraph.chinacloudapi.cn"
Closing this issue for now, since the work to add a new appId will be tracked by #802
